src/tapis-redux/src/store/rootReducer.ts

```diff
@@ -3,14 +3,24 @@ import { authenticator } from '../authenticator/reducer';
 import { systems } from '../systems/reducer';
 import { files } from '../files/reducer';
 import { apps } from '../apps/reducer';
+import { projects } from "../streams/projects/reducer";
 import { jobs } from '../jobs/reducer';
+import { sites } from "../streams/sites/reducer";
+import { instruments } from "../streams/instruments/reducer";
+import { variables } from "../streams/variables/reducer";
+import { measurements } from "../streams/measurements/reducer";
 
 const rootReducer = combineReducers({
   authenticator,
   systems,
   files,
   apps,
-  jobs
+  projects,
```

Comment:
There should be one reducer for streams, with multiple subreducers for the individual components for streams. See the [Projects reducer](https://github.com/tapis-project/tapis-ui/tree/main/src/tapis-redux/src/jobs) that uses a function to aggregate multiple reducers into a single key in the root of the state tree.